### ISSUE TITLE
Technical Design: become_actor syscall

### ISSUE THREAD

**Issue by Stebalien** [filecoin-project/ref-fvm#720]

**Comment by maciejwitowski** [comment]
@raulk do we need the implementation of this to land in Iron? If not, which milestone would it be needed in?

**Comment by raulk** [comment]
Nope, this has actually been descoped from M2.1 entirely based on CPH discussions. Rationale: this provided a pathway for an existing secp256k1 account to be upgraded into an Ethereum-aware account. Based on our current design blueprints, I believe we are segregating accounts so that the Ethereum-aware account will be automatically initialized on a send to an f420... address. I assume that that kind of account won't be able to process native Filecoin transactions (hence why I'm using the term _segregate_), as that would allow reuse of key material for the same signature scheme, but different payload format (which weakens security, something that @Stebalien raised), but worth confirming.
